### PR TITLE
updates for Tsinghua University

### PR DESCRIPTION
**Pull request by chenming-wu**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -4138,6 +4138,7 @@ Hagit Shatkay,University of Delaware,https://www.eecis.udel.edu/~shatkay/,NOSCHO
 Hai Lin 0003,Zhejiang University,http://mypage.zju.edu.cn/0090040,NOSCHOLARPAGE
 Hai Zhao,Shanghai Jiao Tong University,http://bcmi.sjtu.edu.cn/~zhaohai/,4dU5KS0AAAAJ
 Hai Zhou,Northwestern University,http://www.eecs.northwestern.edu/~haizhou/,z8_x7C8AAAAJ
+Hai-Xin Duan,Tsinghua University,https://netsec.ccert.edu.cn/duanhx/,yGTOi34AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:4141]: Is this in the CS department? I can't see evidence that it is.

File: csrankings.csv [hunk 2]
```diff
@@ -5205,6 +5206,7 @@ Jia Xu,York University,http://eecs.lassonde.yorku.ca/faculty/jia-xu/,u8XzSF0AAAA
 Jia-Fu Xu,Nanjing University,https://cs.nju.edu.cn/58/2b/c2639a153643/page.htm,NOSCHOLARPAGE
 Jia-Huai You,University of Alberta,https://webdocs.cs.ualberta.ca/~you/,NOSCHOLARPAGE
 Jia-Ling Wu,National Taiwan University,http://www.forex.ntu.edu.tw/en/members/teacher.php,NOSCHOLARPAGE
+Jiahai Yang,Tsinghua University,http://nmgroup.tsinghua.edu.cn/yang/index_en.htm,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:5209]: Same issue here. Possibly engineering? Please confirm this faculty member meets the inclusion criteria (can solely supervise a PhD student in computer science).

File: csrankings.csv [hunk 3]
```diff
@@ -5248,6 +5250,7 @@ Jianping Wu,Tsinghua University,http://www.tsinghua.edu.cn/publish/ceen/3155/201
 Jianting Zhang,CUNY,http://www-cs.ccny.cuny.edu/~jzhang/,UKjJKUIAAAAJ
 Jianwei Niu 0001,University of Texas at San Antonio,http://www.cs.utsa.edu/~niu/,WURkybsAAAAJ
 Jianwei Yin,Zhejiang University,http://mypage.zju.edu.cn/0001038,0s1A5fwAAAAJ
+Jianwei Zhuge,Tsinghua University,http://netsec.ccert.edu.cn/zhugejw/,ndC7PCQAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:5253]: As above.

File: csrankings.csv (hunk 1 above)
Comment (chenming-wu) [reply csrankings.csv:4141]: In Tsinghua University, we have a school called "School of Information Science and Technology". Several institutes such as CCERT, Institute for Network Sciences and Cyberspace and so on are contained to encourage interdisciplinary collaborations. The faculty members belonging to such institutes can solely supervise a PhD student in computer science.

Here is one of the PhD students whose major is Computer Science in that lab:
http://netsec.ccert.edu.cn/jiangjian/resume

File: csrankings.csv (hunk 2 above)
Comment (chenming-wu) [reply csrankings.csv:5209]: Dr. Yang is a Professor in the Department of Computer Science & Technology, as well as the Institute for Network Sciences and Cyberspace at Tsinghua University, Beijing, China.

Ref: http://nmgroup.tsinghua.edu.cn/yang/index_en.htm#t_1

File: csrankings.csv (hunk 3 above)
Comment (chenming-wu) [reply csrankings.csv:5253]: As above. Similar to Hai-Xin Duan, Jianwei Zhuge is a member of CCERT.

**Pull request merged**